build-constraints.yaml

```diff
@@ -7713,6 +7707,9 @@ github-users:
         - liskin
         - psibi
         - slotThe
+    hakyll:
```

Comment:
This maps from github organizations to users, so it will not have any effect since the repo is under jasper's account.

If you want to have you both listed as maintainers I would change
```
    "Jasper Van der Jeugt @jaspervdj":
        - blaze-html
        - blaze-markup < 0
        - stylish-haskell
        - profiteur
        - psqueues
        - websockets
        - websockets-snap
        - hakyll
```

to
```
    "Jasper Van der Jeugt @jaspervdj":
        - blaze-html
        - blaze-markup < 0
        - stylish-haskell
        - profiteur
        - psqueues
        - websockets
        - websockets-snap
        - hakyll
        
    "Jasper Van der Jeugt @jaspervdj & Alexander Batischev <eual.jp@gmail.com> @Minoru":
        - hakyll        
```